src/components/renderer.tsx

```
@@ -225,7 +229,7 @@ const Renderer = ({ value, image, calendarEvent }: RendererProps) => {
 		return () => {
 			if (container) container.innerHTML = "";
 		};
-	}, [value, image, calendarEvent, isUnifiedMessage]);
+	}, [value, calendarEvent, isUnifiedMessage]);
```

Biome (2.4.14)

[error] isUnifiedMessage changes on every re-render and should not be used as a hook dependency.

(lint/correctness/useExhaustiveDependencies)